src/pages/AdvancedTimers/components/AdvancedTimer.js

```diff
@@ -35,23 +47,30 @@ const AdvancedTimer = ({ timerEntry, startTimer }) => {
     startTimer(timerEntry.timer.name);
   };
 
+  const handleEditTimerName = (e) => {
+    editTimerEntry({
+      variables: {
+        id: timerEntry.id,
+        timerName: e.target.value,
+      },
```

Comment:
```suggestion
      variables: {
        id: timerEntry.id,
        timerName
      },
```
It should be enough since `timerName` should have the same value as `e.target.value`